generators/generator-base-blueprint.js

```diff
@@ -16,6 +16,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const chalk = require('chalk');
```

Comment:
(I don't have the option to put it below so I put this here): the generator has a lot of public methods, are they all tested? Or should some of them be private (or moved to other files because they don't have the same responsibility)?

Reply:
They are all private, there is no reason to use outside.
We only guarantee stability for generator-base and generator-constants.
https://www.jhipster.tech/modules/creating-a-blueprint/

Reply:
Changed to private.